Framework/Algorithms/src/Rebin2D.cpp

```diff
@@ -126,22 +128,23 @@ void Rebin2D::exec() {
   {
     PARALLEL_START_INTERUPT_REGION
 
+    m_progress->report("Computing polygon intersections");
```

Comment:
@SimonHeybrock Yes, I moved it out because the algorithm was spending most of its time writing to std out, the execution speed went down from 160ms to 40ms. 

Is it better to:
- Add a `m_progress->report()` in the inner loop, so that the progress bar scales at the same rate, I tested it and the performance is not significantly affected if there is no string in report('...')
or
- Change size of `nreports` to be the same as `numYbins' (`inputWS->getNumberHistograms()`